### DISCUSSION TITLE
Tambah devices via Postman

### DISCUSSION THREAD

**Discussion by danielraysa** [kirimwa-id/forum#74] [Tanya Jawab] [answered]
Mohon ijin bertanya, apakah kita bisa melakukan request API via Postman?
Saya coba lakukan post request ke https://api.kirimwa.id/v1/devices sesuai dengan referensi quick intro dari https://developer.kirimwa.id dan hasilnya seperti berikut:
`"message": "Error: Parameter device_id is required."`
Sudah saya tambahkan `device_id` nya namun seperti gambar berikut, namun hasilnya tetap sama
![image](https://user-images.githubusercontent.com/24839306/145165617-7e1a0945-b77b-4301-add6-4b7114f151ef.png)

Terimakasih

**Comment by rioastamal** [comment, marked as answer]
Content-Type yang harus dikirim adalah `application/json` dan body

```json
{ "device_id": "DEVICE_ID" }
```

**Reply by danielraysa** [reply]
Setelah dicoba menggunakan body `form-data` munculnya `"message": "Bad JSON input."`.Tapi ketika mencoba body `raw` dan ditulis sesuai format di atas, baru bisa muncul hasilnya. Terimakasih banyak untuk jawaban/bantuannya
